### PR TITLE
op-guide: improve docker deployment

### PR DESCRIPTION
**Pull request by iamxy**
@siddontang @shenli @tennix PTAL

### CODE REVIEW

File: op-guide/docker-deployment.md [hunk 1]
```diff
@@ -17,164 +19,179 @@ docker pull pingcap/pd:latest
...
-Query OK, 0 rows affected (0.01 sec)
+The TiKV and PD can be started with a specified configuration file, which includes some advanced parameters, for the performance tuning.
 
-mysql> USE mydb;
-Database changed
+Assume that the path to configuration file of PD and TiKV on the host is `/path/to/config/pd.toml` and `／path/to/config/tikv.toml`
 
-mysql> CREATE TABLE mytable ( id INT, data VARCHAR(100), dt DATE, PRIMARY KEY (id) );
-Query OK, 0 rows affected (0.01 sec)
+You can start TiKV and PD as following
```
Comment (siddontang) [inline op-guide/docker-deployment.md]: as following:

**Comment by iamxy** [comment]
@queenypingcap PTAL

File: op-guide/docker-deployment.md [hunk 2]
```diff
@@ -17,164 +19,179 @@ docker pull pingcap/pd:latest
...
-Query OK, 0 rows affected (0.01 sec)
+The TiKV and PD can be started with a specified configuration file, which includes some advanced parameters, for the performance tuning.
 
-mysql> USE mydb;
-Database changed
+Assume that the path to configuration file of PD and TiKV on the host is `/path/to/config/pd.toml` and `／path/to/config/tikv.toml`
 
-mysql> CREATE TABLE mytable ( id INT, data VARCHAR(100), dt DATE, PRIMARY KEY (id) );
-Query OK, 0 rows affected (0.01 sec)
+You can start TiKV and PD as following:
```
Comment (queenypingcap) [inline op-guide/docker-deployment.md]: following -> follows

**Comment by iamxy** [comment]
rest PTAL @siddontang @queenypingcap

**Comment by queenypingcap** [comment]
LGTM

**Comment by tennix** [comment]
LGTM

**Pull request merged**